analysis-engine/src/main/java/org/sonarsource/sonarlint/core/analysis/sonarapi/MapSettings.java

```diff
@@ -265,9 +265,7 @@ public Double getDouble(String key) {
         return ArrayUtils.EMPTY_STRING_ARRAY;
       }
 
-      return Arrays.stream(value.split(",", -1)).map(String::trim)
-        .map(s -> s.replace("%2C", ","))
```

Comment:
We were doing some kind of URL decode here, and I think we don't do it anymore, was that important ?

Reply:
AFAIK this is a remain of a very very old bug we had in the UI. I don't think we need to keep that anymore.